js/common/util/format_date.js

```diff
@@ -0,0 +1,19 @@
+// Check for support of native locale string options
+function toLocaleDateStringSupportsOptions() {
+  try {
+    new Date().toLocaleDateString('i');
+  } catch (e) {
+    return e.name === 'RangeError';
+  }
+  return false;
+}
+
+export default (toLocaleDateStringSupportsOptions() ?
+  function (dateObj, locales) {
+    return dateObj.toLocaleDateString(
+      locales,
+      { year: 'numeric', month: 'long', day: 'numeric' },
+    );
+  } : function (dateObj, locales) { // eslint-disable-line no-unused-vars
+    return window.fecha.format(dateObj, 'mediumDate');
```

Comment:
You could also choose to expose fecha on the window from our JS.

```js
import fecha from 'fecha';
window.fecha = fecha;
```

Or, if all functions that use fecha are ported to JS, we can not expose it at all.

Reply:
That's what I tried, but unfortunately there's some weird binding to `this` upstream, that was prevent it from loading